test/vanilla/AcceptanceTests/acceptanceTests.ts

```diff
@@ -944,19 +944,19 @@ describe('nodejs', function () {
           should.exist(result);
           var date = result;
           done();
-          //TODO: NodeJS doesn't deserialize this time correctly
-          var dateFormat = 'ddd, DD MMM YYYY HH:mm:ss';
-          var myMoment = moment.utc('Mon, 01 Jan 0001 00:00:00 GMT', dateFormat);
-          should.not.exist(myMoment.toDate().toUTCString());
-
-          date.getUTCFullYear().should.equal(1);
-          date.getUTCMonth().should.equal(0);
-          date.getUTCDate().should.equal(1);
-          date.getUTCHours().should.equal(0);
-          date.getUTCMinutes().should.equal(0);
-          date.getUTCSeconds().should.equal(0);
-          date.getUTCMilliseconds().should.equal(0);
-          done();
+          // //TODO: NodeJS doesn't deserialize this time correctly
```

Comment:
please use `this.skip()`

Reply:
Or just remove the code entirely?

Reply:
For reference here's what the impl of this test looks like in autorest.typescript
```js
it('should get UTC min dateTimeRfc1123 value', function (done) {
  testClient.datetimerfc1123.getUtcMinDateTime(function (error, result) {
    should.not.exist(error);
    should.exist(result);
    // Parsing the minimum date 'Mon, 01 Jan 0001 00:00:00 GMT' doesn't
    // work properly in nodejs, so we'll just test that the result exists
    done();
  });
});
```

Perhaps it should go away entirely, but I also don't like it showing up in the coverage report if there's no way for us to ever pass it properly..

Reply:
hmmm will simply go with `it.skip` for now, that way it won't cheat on the coverage and the code is still there